api/views.py

```diff
@@ -1,3 +1,56 @@
-from django.shortcuts import render
+from django.http import JsonResponse
+from rest_framework_simplejwt.tokens import RefreshToken
 
+from .serializers import *
+from .models import *
+from rest_framework import viewsets, views
 # Create your views here.
+
+
+class UserViewSet(viewsets.ModelViewSet):
+    serializer_class = UserSerializer
+    queryset = User.objects.all()
+
+
+class SignUpView(views.APIView):
+    serializer_class = SignUpSerializer
+
+    def post(self, request):
+        serializer = self.serializer_class(data=request.data)
+
+        if serializer.is_valid():
+            user = serializer.save()
+
+            token = RefreshToken.for_user(user)
+            refresh = str(token)
+            access = str(token.access_token)
+
+            return JsonResponse({
+                'user': user.username,
+                'refresh': refresh,
+                'access': access,
+            })
+        else:
+            return JsonResponse(serializer.errors)
+
+
+class LoginView(views.APIView):
+    serializer_class = LoginSerializer
+
+    def post(self, request):
+        serializer = self.serializer_class(data=request.data)
+
+        if serializer.is_valid():
+            user = serializer.validated_data['user']
+            refresh = serializer.validated_data['refresh']
+            access = serializer.validated_data['access']
+
+            return JsonResponse({
+                'user': user.username,
+                'is_voted': user.is_voted,
+                'refresh': refresh,
+                'access': access,
+            })
```

Comment:
```suggestion
            return JsonResponse({
                'message': '로그인 성공',
                'user': user.username,
                'is_voted': user.is_voted,
                'access': access,
                'refresh': refresh,
            })
```

여기도 마찬가지로 이렇게 응답해주면 더 좋지 않을까 싶습니다!!